src/games/strategy/triplea/ai/proAI/util/ProAttackOptionsUtils.java

```diff
@@ -445,23 +458,29 @@ public void findDefendOptions(final PlayerID player, final List<Territory> myUni
         ProMatches.territoryHasNoEnemyUnitsOrCleared(player, data, clearedTerritories), clearedTerritories, false,
         false);
     findLandMoveOptions(player, myUnitTerritories, moveMap, unitMoveMap, landRoutesMap,
-        Matches.isTerritoryAllied(player, data), new ArrayList<Territory>(), false, false, false);
+        Matches.isTerritoryAllied(player, data), new ArrayList<Territory>(), new ArrayList<Territory>(), false, false,
+        false);
     findAirMoveOptions(player, myUnitTerritories, moveMap, unitMoveMap,
-        ProMatches.territoryIsNotConqueredAlliedLand(player, data), new ArrayList<Territory>(), false, false, false);
+        ProMatches.territoryIsNotConqueredAlliedLand(player, data), new ArrayList<Territory>(),
+        new ArrayList<Territory>(), false, false, false);
     findAmphibMoveOptions(player, myUnitTerritories, moveMap, transportMapList, landRoutesMap,
         Matches.isTerritoryAllied(player, data), new ArrayList<Territory>(), false, false, false);
   }
 
-  public void findMaxEnemyAttackUnits(final PlayerID player, final List<Territory> myConqueredTerritories,
+  public void findEnemyAttackOptions(final PlayerID player, final List<Territory> myConqueredTerritories,
```

Comment:
Check to see if you can use a Set instead of List as parameter types. You convert a lot of sets into ArrayLists when calling this function, looks like that is more common than passing in an actual list.